pkg/image/api/types.go

```diff
@@ -194,6 +194,19 @@ type ImageStreamTag struct {
 	unversioned.TypeMeta
 	kapi.ObjectMeta
 
+	// Tag is the spec tag associated with this image stream tag, and it may be null
+	// if only pushes have occured to this image stream.
+	Tag *TagReference
```

Comment:
No, this is the spec tag

On Thu, Feb 11, 2016 at 8:50 AM, David Eads <notifications@github.com>
wrote:

> In pkg/image/api/types.go
> <https://github.com/openshift/origin/pull/6925#discussion_r52602636>:
>
> > @@ -194,6 +194,19 @@ type ImageStreamTag struct {
> >  	unversioned.TypeMeta
> >  	kapi.ObjectMeta
> >
> > +	// Tag is the spec tag associated with this image stream tag, and it may be null
> > +	// if only pushes have occured to this image stream.
> > +	Tag *TagReference
>
> Are these mutually exclusive to Image?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6925/files#r52602636>.
>